### PR TITLE
Fix Eady turbulence example

### PR DESCRIPTION
**Pull request by navidcy**
I couldn't run docs locally to make sure all is good...

running 
```bash
julia --project=docs/ -e 'using Pkg; Pkg.instantiate(); Pkg.develop(PackageSpec(path=pwd()))'; julia --project=docs/ docs/make.jl;
```

returns

```bash
...
[ Info: Expanding citation: Kundu15.
[ Info: CheckDocument: running document checks.
[ Info: Populate: populating indices.
ERROR: LoadError: `makedocs` encountered an error. Terminating build
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] runner(::Type{Documenter.Builder.RenderDocument}, ::Documenter.Documents.Document) at /Users/navid/.julia/packages/Documenter/pjwqp/src/Builder.jl:255
 [3] dispatch(::Type{Documenter.Builder.DocumentPipeline}, ::Documenter.Documents.Document) at /Users/navid/.julia/packages/Documenter/pjwqp/src/Utilities/Selectors.jl:167
 [4] #2 at /Users/navid/.julia/packages/Documenter/pjwqp/src/Documenter.jl:241 [inlined]
 [5] cd(::Documenter.var"#2#3"{Documenter.Documents.Document}, ::String) at ./file.jl:104
 [6] #makedocs#1 at /Users/navid/.julia/packages/Documenter/pjwqp/src/Documenter.jl:240 [inlined]
 [7] top-level scope at /Users/navid/Research/Oceananigans.jl/docs/make.jl:141
 [8] include(::Function, ::Module, ::String) at ./Base.jl:380
 [9] include(::Module, ::String) at ./Base.jl:368
 [10] exec_options(::Base.JLOptions) at ./client.jl:296
 [11] _start() at ./client.jl:506
in expression starting at /Users/navid/Research/Oceananigans.jl/docs/make.jl:141
```

which I couldn't trace down...

### CODE REVIEW

**Comment by navidcy** [comment]
Unfortunately a docs preview was not pushed (because I don't have the appropriate keys setup on my fork) and also I cannot build docs locally.

@ali-ramadhan any ideas why can't I build docs locally?

**Comment by glwagner** [comment]
I can't figure out the error. But I usually just run

```
julia --project make.jl
```

The line you're using is a bit more complicated --- do you just need to instantiate the environment?

**Comment by glwagner** [comment]
I think you can build just one example by running https://github.com/CliMA/Oceananigans.jl/blob/master/docs/make_example.jl. eg `julia --project make_example.jl`

**Comment by navidcy** [comment]
> The line you're using is a bit more complicated --- do you just need to instantiate the environment?

The line I am using is simply what `.travis.yml` calls and the "complication" you are referring to probably is exactly the instantiation of the environment :)

**Comment by navidcy** [comment]
built docs locally... looks good!

Screen Shot 2020-10-17 at 5 56 04 pm

File: examples/eady_turbulence.jl [hunk 1]
```diff
@@ -60,18 +60,22 @@
...
+# to $\boldsymbol{u}_h |\boldsymbol{u}_h|$, where 
+# $\boldsymbol{u}_h = u \boldsymbol{\hat{x}} + v \boldsymbol{\hat{y}}$ is the horizontal velocity.
 #
 # The $x$-component of the quadratic bottom drag is thus
-#
-# $ τ_{xz}(z=L_z) = - cᴰ u \sqrt{u^2 + v^2} \, , $
+# 
+# ```math
+# \tau_{xz}(z=L_z) = - c^D u \sqrt{u^2 + v^2} \, ,
+# ```
```
Comment (glwagner) [inline examples/eady_turbulence.jl:70]: a math block... genius!

**Comment by glwagner** [comment]
> the "complication" you are referring to probably is exactly the instantiation of the environment :)

Ah for sure --- I instantiate as needed instead, which is rare because the environment doesn't change that often. I'm not sure what `Pkg.develop(PackageSpec(path=pwd()))` is needed for either; I don't do this and local builds seem to work, but maybe I should start?

**Comment by navidcy** [comment]
Well I'm not exactly sure if `.develop()` is needed. I just copied this from `.travis.yml`.
However, when later I ran the same exact line the above error didn't appear...

**Comment by navidcy** [comment]
Travis below seems to be queued but if you click on the travis link it shows that everything run ok... weird...

**Comment by navidcy** [comment]
https://travis-ci.com/github/CliMA/Oceananigans.jl/builds/190625652

**Comment by navidcy** [comment]
ok @glwagner this is ready for review

File: examples/eady_turbulence.jl (hunk 1 above)
Comment (glwagner) [reply examples/eady_turbulence.jl:70]: still genius

File: examples/eady_turbulence.jl [hunk 2]
```diff
@@ -412,7 +416,7 @@ anim = @animate for (i, iter) in enumerate(iterations)
            size = (1000, 800),
            link = :x,
          layout = Plots.grid(2, 2, heights=[0.5, 0.5, 0.2, 0.2]),
-          title = [@sprintf("ζ/f(t=%s) (s⁻¹)", prettytime(t)) @sprintf("δ(t=%s) (s⁻¹)", prettytime(t)) "" ""])
+          title = [@sprintf("ζ(t=%s)/f", prettytime(t)) @sprintf("δ(t=%s) (s⁻¹)", prettytime(t)) "" ""])
```
Comment (glwagner) [inline examples/eady_turbulence.jl:419]: quite :-P

File: examples/eady_turbulence.jl [hunk 3]
```diff
@@ -60,18 +60,22 @@
 # the flux is negative (downwards) when the velocity at the bottom boundary is positive, and 
 # positive (upwards) with the velocity at the bottom boundary is negative.
 # This drag term is "quadratic" because the rate at which momentum is removed is proportional
-# to $\bm{u}_h \sqrt{u^2 + v^2}$, where $\bm{u}_h = u \bm{\hat x} + v \bm{\hat{v}$ is the
-# horizontal velocity.
+# to $\boldsymbol{u}_h |\boldsymbol{u}_h|$, where 
+# $\boldsymbol{u}_h = u \boldsymbol{\hat{x}} + v \boldsymbol{\hat{y}}$ is the horizontal velocity.
```
Comment (glwagner) [inline examples/eady_turbulence.jl:64]: this is fine and I will merge, but just out of curiosity why boldsymbol vs bm?

**Review by glwagner — APPROVED** [review]

**Comment by glwagner** [comment]
Thank you very much @navidcy, beautifier of examples and docs and unsung hero to confuzzled scientists round the world.

File: examples/eady_turbulence.jl (hunk 3 above)
Comment (navidcy) [reply examples/eady_turbulence.jl:64]: hm... I thought `\bm` was part of the problem for not latexifying... perhaps I was wrong?

**Comment by navidcy** [comment]
🙏🏼

File: examples/eady_turbulence.jl (hunk 3 above)
Comment (glwagner) [reply examples/eady_turbulence.jl:64]: I think your math block fixed the main problem but I could be wrong. Used to be that KaTeX / Documenter didn't support `\boldsymbol` (_only_ `\bm` was supported). Seems that's no longer the case, though if I had to guess, I would guess that they are aliases for each other. I believe `\bm` is supposed to be some kind of simple, lightweight version of `\boldsymbol`.

Comment (navidcy) [reply examples/eady_turbulence.jl:64]: The above assertion can be tested... but I'll leave that for some future time :)

Comment (ali-ramadhan) [reply examples/eady_turbulence.jl:64]: They might be aliases in KaTeX/Documenter but I think `bm` is a stand-alone package in vanilla LaTeX: https://ctan.org/pkg/bm?lang=en

**Pull request merged**